### PR TITLE
Use device libs that have proper shfl implementation for xla

### PR DESCRIPTION
**Pull request by sumesh13**
Point deb package to location with latest device libs as well

### CODE REVIEW

File: tensorflow/tools/ci_build/Dockerfile.rocm [hunk 1]
```diff
@@ -100,8 +100,9 @@ RUN bash -c 'echo -e "gfx803\ngfx900\ngfx906" >> /opt/rocm/bin/target.lst'
 #  - rocblas-2.0.1.0-Linux.deb
 #  - rocfft-0.8.9.0-Linux.deb
 #  - rocrand-1.8.2-Linux.deb
-RUN cd $HOME && wget https://www.dropbox.com/s/e6yh6glemrkoo91/tf-pr318-deb-pkgs.tar.gz && \
-    tar xzf tf-pr318-deb-pkgs.tar.gz && dpkg -i ./tf-pr318-deb-pkgs/*.deb
+# Grab deb packages with pr318 and shfl implementation used by xla
+RUN cd $HOME && wget https://www.dropbox.com/s/z8b4ek3z0k7ir8v/tf-pr318-shfl-deb-pkgs.tar.gz?dl=0 && \
```
Comment (whchung) [inline tensorflow/tools/ci_build/Dockerfile.rocm]: @sumesh13  please remove `dl=0`. please refer to:

https://www.dropboxforum.com/t5/Installation-and-desktop-app/dl-0-and-dl-1-does-the-same-thing/td-p/170029

Comment (whchung) [reply tensorflow/tools/ci_build/Dockerfile.rocm]: or use `dl=1`. refer to:
https://help.dropbox.com/en-us/desktop-web/force-download

**Comment by sumesh13** [comment]
retest this please

**Comment by whchung** [comment]
jenkins: retest this please

**Comment by whchung** [comment]
jenkins: retest this please

(doesn't know why `rocm-xla` target was aborted)

**Comment by sumesh13** [comment]
I see the new  dropbox contents being used in the rocm path ..the xla path is getting aborted repeatedly

**Comment by whchung** [comment]
From the previous log it seems it was aborted after running for 180 minutes. Need to figure out how to relax this restriction.

**Comment by whchung** [comment]
just to catch the current progress on `rocm-xla` path:

http://ml-ci.amd.com:21096/job/tensorflow-upstream-unit-tests-rocm-xla/57/console

```
[32m[20,338 / 20,370][0m 384 / 459 tests, [31m[1m25 failed[0m;[0m [Sched] Testing //tensorflow/compiler/xla/tests:convolution_dimension_numbers_test_cpu; 297s ... (32 actions, 1 running)
```

looks a bit better than build 44 but let's wait until the whole build completes.

**Pull request merged**